src/components/dashboard/constants.js

```diff
@@ -46,4 +46,7 @@ export const XS_NUM_COLUMNS = 1;
 export const XS_PIXELS = 0;
 
 export const USER_SURVEY = "https://cyver.se/srvy";
-export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
\ No newline at end of file
+export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
+export const FEATURE_MATRIX_URL = "https://cyverse.org/subscribe";
+export const CHECKOUT_URL =
+    "https://cyverse-subscription-sandbox.phoenixbioinformatics.org";
```

Comment:
This is sandbox for testing Phx Bio Credit card payment site.

Reply:
Good point. But as far as I know, the `Buy` button here is only for QA env to test. For prod, the `Buy` button will be in main website subscription page  (https://cyverse.org/subscribe)(currently being built by Mariah). So I will delete the `Buy` button once that page is live.